opm/parser/eclipse/EclipseState/Schedule/OilVaporizationProperties.hpp

```diff
@@ -0,0 +1,39 @@
+#ifndef DRSDT_HPP
+#define DRSDT_HPP
+#include <iostream>
+#include <string>
+#include <memory>
+#include <opm/parser/eclipse/EclipseState/Schedule/ScheduleEnums.hpp>
+
+namespace Opm
+{
```

Comment:
> in particular why (or if?) behaviour is exclusive in some sense

Yes - the behavior is exclusive; any one of the three keywords `{VAPPARS, DRSDT, DRVDT}`will cancel previous settings of the other keywords.